.github/PULL_REQUEST_TEMPLATE/icon_addition.md

```diff
@@ -16,8 +16,6 @@ App Name (`com.package.app` → `drawable.svg`)
 App Name (`com.package.app`)
 App Name (`com.package.app`)
 
-## Contributor's checklist
+## Contributor's check
 <!-- Replace [ ] with [x] to check -->
-- [ ] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md)
-- [ ] I have ensured that Lawnicons builds correctly
-- [ ] I am willing to make changes to my icons if someone suggests changes
+- [ ] I made the icons following [the Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) (upd. Jan 2024) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.
```

Comment:
why do we have to say upd jan 2024

Reply:
I marked when the last guidelines update was, so old contributors would be aware about changes.

Reply:
I'd say keep the updated message for max a year in there and then remove